shared/contracts/BaseTemplate.sol

```diff
@@ -272,15 +274,15 @@ contract BaseTemplate is APMNamehash, IsContract {
     /* APPS */
 
     function _installNonDefaultApp(Kernel _dao, bytes32 _appId) internal returns (address) {
-        return _installApp(_dao, _appId, new bytes(0), false);
+        return _installNonDefaultApp(_dao, _appId, new bytes(0));
     }
 
     function _installNonDefaultApp(Kernel _dao, bytes32 _appId, bytes _data) internal returns (address) {
```

Comment:
Could we rename `_data` to `_initializeData` here and elsewhere to make it more clear?

Reply:
+1